### DISCUSSION TITLE
NotImplementedError. Could not find signature for dtype

### DISCUSSION THREAD

**Discussion by a-wyatt** [ibis-project/ibis#5244] [Q&A] [answered]
Hi all,

I am new to Ibis but really like it so far, great job!
I am using a MSSQL backend and hitting an error.
`NotImplementedError: Could not find signature for dtype: <MSDialect_pymssql, UNIQUEIDENTIFIER>`

There are two columns in the table that have an MSSQL type of uniqueidentifier.
It seems reasonable to parse these as strings.

Is there any suggested method for extending support for types like this or cast to string.
The issue seems to be that the first thing I need to do is create a handler to the table which fails before I can do something like this...
```
t = con.table('t')
# NotImplementedError here
# Can't do a cast first
t.uuid.cast('str')
```
Any help would be appreciated.

**Comment by cpcloud** [comment, marked as answer]
Hey @a-wyatt thanks for opening this discussion! This looks like a bug, so I'm going to convert it to an issue and we can continue the conversation there.
